integration-tests/gradle/src/integrationTest/kotlin/org/jetbrains/dokka/it/gradle/kotlin/StdlibGradleIntegrationTest.kt

```diff
@@ -46,6 +46,7 @@ class StdlibGradleIntegrationTest(override val versions: BuildVersions) : Abstra
     }
 
     @Test
+    @Ignore
```

Comment:
Since we lost backwards compatibility with KGP <= 1.6.21, this test is broken as it's still using Kotlin 1.4.

If we don't do anything to preserve backwards compatibility, I would actually like to nuke this test and the submodule. Am I right in understanding that [kotlin-dokka-stdlib](https://github.com/JetBrains/kotlin-dokka-stdlib) repo is deprecated and will/should not be updated?